components/RoadmapPage/RoadmapPage.tsx

```diff
@@ -14,13 +14,16 @@ const RoadmapPage = () => {
   const stages: StageData[] = data as StageData[]
 
   const [currentIndex, setCurrentIndex] = useState(0)
-  const [selectedIndex, setSelectedIndex] = useState(0)
+  const [activeIndex, setActiveIndex] = useState(0)
+  const [hoveredIndex, setHoveredIndex] = useState(0)
   const [swiperCtrl, setSwiper] = useState<any>()
 
   const isResponsive = useMediaQuery("(max-width: 1150px)")
   const isMobile = useMediaQuery("(max-width: 768px)")
   const isIphone = useMediaQuery("(max-width: 390px)")
 
+  const changeHoverIndex = (hoverdIndex: number) => setHoveredIndex(hoverdIndex)
```

Comment:
- typo in `hoverdIndex`.
- already a variable called `hoveredIndex`.
- new suggestion below. feel free to come up with a new name

```suggestion
  const changeHoverIndex = (newIndex: number) => setHoveredIndex(newIndex)
```